apl-core/src/test/resources/db/currency_founder-data.sql

```diff
@@ -1,13 +1,14 @@
-delete from currency_founder;
+delete
+from currency_founder;
 
 INSERT INTO currency_founder
-(DB_ID, CURRENCY_ID,            ACCOUNT_ID,         AMOUNT, HEIGHT, LATEST, DELETED) VALUES
-(9,      6847297283087791598, -6392448561240417498,  100000, 98790,  false, false),
-(10,     6847297283087791598, -6392448561240417498,  100000, 98880,  true,  false),
-(16,     7766734660591439874, 2650055114867906720,   100000, 98982,  false, false),
-(19,     7766734660591439874, 2650055114867906720,   100000, 98983,  false, false),
-(20,     7766734660591439874, 2650055114867906720,   100000, 98990,  true,  false),
-(26,     2582192196243262007, 5122426243196961555,   100000, 99991,  false, false),
-(29,     2582192196243262007, 5122426243196961555,   100000, 99992,  false, false),
-(30,     2582192196243262007, 5122426243196961555,   100000, 99999,  true,  false)
+    (DB_ID, CURRENCY_ID, ACCOUNT_ID, AMOUNT, HEIGHT, LATEST, DELETED)
```

Comment:
Such format decrease readability and data matching, since header does not follow the content columns